src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -197,8 +224,9 @@ protected function edit_post_link( $linked_post_id ) {
 	/**
 	 * Determine if the event can use the default setting
 	 *
-	 * @param array $current_organizers
+	 * @since ??
 	 *
+	 * @param array $current_organizers
```

Comment:
Similar to elsewhere, if we're changing the docblock and are touching this line anyway, let's make it accurate: `@param array $current_posts`